src/main/operatingSystems.ts

```diff
@@ -5,7 +5,7 @@ const OS = {
     Mac: 'darwin',
   }
   
-  function byOS(handlers) {
+  function byOS(handlers: any) {
```

Comment:
Pretty sure this is all useless code FYI, inherited from the obsRecorder copy paste.

Reply:
Oh, yeah it's there to basically support MacOS as well from what I can gather. If you don't plan on supporting that, we can remove it - if you want to keep the support in (for that part of the code anyway) then we should keep it.